.gitignore

```diff
@@ -1,3 +1,4 @@
 ./.DS_Store
 ./spec/.rspec
 coverage
+./lib/shop(old).rb
```

Comment:
What is the role of this file? Is it a temporary file created by your editor? or some "backup" file? You don't need a backup using git.